src/library/Graphs/DistributionDoughnut.tsx

```diff
@@ -0,0 +1,95 @@
+// Copyright 2023 @paritytech/polkadot-staking-dashboard authors & contributors
```

Comment:
Striving for a shorter name: `GeoDonut`?

Reply:
Sounds good to me... Applying the change.